CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - `oe_random()` now depends on the hardware-based source of RNG instead of cryptography libraries.
+- Ocall stack-stitching implemented as per Debugging Contract. OE SDK performs stack stitching
```

Comment:
Nit: Casing consistency "OCall"

Side note: we're referring to the "Debugging Contract" as a proper noun here, but I don't believe it's officially documented anywhere in the repo today (maybe as code comments). Might be good to link this to some description of the contract (e.g. debugger/README.md or a design doc perhaps?) when you tackle the documentation TODO.

Reply:
I have created a documentation task to publish the contract. The contract will be uploaded after the next PR related to debuggers goes in as well.
#2619 2619